test_mongo.py

```diff
@@ -0,0 +1,69 @@
+import os
+import dotenv
+import pytest
+import requests
+import app as app
+import mongo_items as mongo
+from threading import Thread 
+import time
+import unittest
+from selenium import webdriver
+from webdriver_manager.chrome import ChromeDriverManager
+from selenium.webdriver.remote.webelement import WebElement
+from selenium.webdriver.common.keys import Keys
+
+def test_task_journey(driver, test_app):
+    driver.get('http://localhost:5000/')
+    
+    task_id = test_app.mongo_items.add_item("test task")
+    moved_to_inprogress = test_app.mongo_items.inprogress_item(task_id)
+    mark_as_done = test_app.mongo_items.markAsDone(task_id)
+
+    assert driver.title == 'To-Do App'
+    assert task_id is not None
+    assert moved_to_inprogress is True
+    assert mark_as_done is True
+
+def test_create_and_delete_board():
+     database_id = mongo.create_database('test-db')
+     database_deleted = mongo.delete_database('test-db')
+
+     assert database_id is not None
+     assert database_deleted is True
+    
+
+@pytest.fixture(scope='module')
+def test_app():
+    # Create the new board & update the board id environment variable
+    board_id = mongo.create_database("TestAppBoard") 
```

Comment:
Here you create and delete your test database, but you don't actually override the environment variable for the database name, so the `test_createTask` test still saves to your live board!

We're also not loading the `.env` file anywhere, which means that this test fails unless the variables are explicitly set on the environment beforehand (as travis does)